### PR TITLE
Sitemap plugin

### PR DESCRIPTION
**Pull request by dfoulks1**

### CODE REVIEW

File: bin/local-pelican-site.sh [hunk 1]
```diff
@@ -73,7 +73,8 @@ then
   cd $IP && git pull > /dev/null && cd .. 
 else
   echo "Cloning $IP"
-  git clone $GH/$IP 2>&1 
+  # Sometimes useful to add -b <branch> for buildsite testing
+  git clone -b sitemap-plugin $GH/$IP 2>&1
```
Comment (dave2wave) [inline bin/local-pelican-site.sh:77]: This may be true for your development, but is certainly not true in production,

**Review by dave2wave — COMMENTED** [review]
Other than the branching issue this looks ok to me.

File: bin/local-pelican-site.sh (hunk 1 above)
Comment (dfoulks1) [reply bin/local-pelican-site.sh]: resolved here:https://github.com/apache/infrastructure-pelican/pull/39/commits/a96c78cbb6a590816da1df0f059a8fbd0c2d15d3

**Review by dave2wave — APPROVED** [review]
LGTM - nice and concise

File: bin/pelican.auto.ezt [hunk 2]
```diff
@@ -72,6 +72,24 @@ ASF_GENID = {
...
 [end]
+
+[if-any uses_sitemap]
+SITEMAP = {
+    "exclude": [sitemap.exclude],
+    "format": "[sitemap.format]",
+    "priorities": {
+        "articles": [sitemap.priorities.articles],
+        "indexes": [sitemap.priorities.indexes],
+        "pages": [sitemap.priorities.pages]
```
Comment (gstein) [inline bin/pelican.auto.ezt:83]: comma after each item is best-practice so that adding a item doesn't require modifying the prior line.

**Comment by gstein** [comment]
LGTM. Merge away.

**Pull request merged**